ntoskrnl/mm/i386/procsup.c

```diff
@@ -0,0 +1,98 @@
+/*
+ * PROJECT:         ReactOS Kernel
+ * LICENSE:         BSD-3-Clause (https://spdx.org/licenses/BSD-3-Clause.html)
+ * FILE:            ntoskrnl/mm/i386/procsup.c
```

Comment:
Same thing with the header here. Btw, is `BSD-3-Clause` your license of choice or it's just a leftover from somewhere?

Reply:
Everything in ARM3 is "BSD", which is not what I'd call precise. BSD-3-clause is what I consider as the closest of what this should be : do whatever you want with that, but don't call my name if all hell breaks loose.